### ISSUE TITLE
Experiment user count doesn't always update after enabling experiment

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#583]
Found in http://testpilot.dev.mozaws.net/__version__ (a41e42424096277f7fa0504baf5bca046b9ae346):

``` json
{ "version": "",
  "commit": "a41e42424096277f7fa0504baf5bca046b9ae346",
  "source": "https://github.com/mozilla/testpilot"}
```

Addon: Test Pilot @ 0.4.13-dev-a41e424

---

I'm signed in using my @mozilla account, and before enabling the add-on, there are 5 active users.
Not sure if this is because I've used my @mozilla account previously and it's deduping me.

**Before:**
![firefox_test_pilot_-_activity_stream_and_new_issue_ _mozilla_testpilot_and_add-ons_manager](https://cloud.githubusercontent.com/assets/557895/14394661/2fa6d006-fd82-11e5-8105-2d146265a461.png)

**Enabling:**
![firefox_test_pilot_-_activity_stream_and_new_issue_ _mozilla_testpilot_and_add-ons_manager](https://cloud.githubusercontent.com/assets/557895/14394684/51a23100-fd82-11e5-8a61-86e7d9973db4.png)

**After:**
![firefox_test_pilot_-_activity_stream_and_new_issue_ _mozilla_testpilot_and_add-ons_manager](https://cloud.githubusercontent.com/assets/557895/14394693/6270dda6-fd82-11e5-96b3-71e63e4e438e.png)

**Comment by pdehaan** [comment]
Similarly, disabling the experiment doesn't reduce the number of active users.

**Comment by pdehaan** [comment]
I'm wondering if this is because I am signed in as the same user (my mozilla.com email) in different browsers/profiles, so it isn't counting me as a unique install/user, despite the fact I personally have it installed in 4-5 times).

**Comment by ckprice** [comment]
Hmm, getting some inconsistency here on http://testpilot.dev.mozaws.net/

Tried once on DevEdition, and the count on Universal Search had an odd flicker where it went up, then down real quick.

Went back again on Beta, and everything worked okay on all experiments.

I don't think we have to call this as a blocker for launch. I'm adding it to the post-launch milestone so we keep an eye on it.

**Comment by lmorchard** [comment]
As currently implemented, [this is a count of distinct users with installations](https://github.com/mozilla/testpilot/blob/master/testpilot/experiments/models.py#L62). That is, if you have installed the experiment in 5 different browser profiles as the same user, that will count as 1 for this number.

However, if you uninstall 1 but still have 4 other installations in the database - that will still count as 1 for you. You have to have _0 installations in any browser profile associated with your user_ for this count to go down. 

If that's too confusing, maybe we should change that model property to just count installations instead of distinct users?

As for the flickering, that's because we do [a little client-side prediction](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/views/experiment-page.js#L225): When you hit the install button, we increment the install count immediately and then refresh the model data for the experiment. But, it looks like if you already have been counted as a user with an install, this count will appear to decrement because the model data disagrees with the prediction. 

Maybe we should also get rid of this client-side counter increment?

**Comment by pdehaan** [comment]
Closing. I believe this was fixed by #661.
I'm seeing the counts updating on stage+production currently.
